### PR TITLE
Fix CAP bug

### LINKED ISSUE #142: Fix bugs in CAP Class
- CAP Class Message in AddCommand and EditCommand 
- Error thrown when we do edit 1 cap/5

### PR DESCRIPTION
**Pull request by maryjess**
Changes made:
- Edit format in `SUCCESS_MESSAGE` of both `AddCommand` and `EditCommand` to be more specific from `cap/CAP` to `cap/CAP_VALUE/MAX_CAP_VALUE`
- Catch `ArrayIndexOutOfBoundsException` that was thrown when input for CAP field was only given one value e.g. `cap/2.0`
- Add test in `ParserUtilTest.java` for invalid CAP value given (only one value)

Resolves #142.

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 1]
```diff
@@ -152,7 +152,7 @@ public static Cap parseCap(String cap) throws ParseException {
                 throw new ParseException(Cap.MESSAGE_CONSTRAINTS);
             }
             return new Cap(capValue, maximumCapValue);
-        } catch (NumberFormatException e) {
+        } catch (NumberFormatException | ArrayIndexOutOfBoundsException e) {
             throw new ParseException(Cap.MESSAGE_CONSTRAINTS);
```
Comment (lulucopter) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java:156]: Do we want to have a different error message when a maximum value is not given?

**Review by lulucopter — COMMENTED** [review]
Overall LGTM. Want to get your opinions on having a different message thrown when a cap is input without a maximum value (I think will be a common user mistake)

**Review by deeyonn — APPROVED** [review]
Consider the point @lulucopter made, else it LGTM

**Review by JasonCP14 — APPROVED** [review]
LGTM

**Pull request merged**